editor/src/clj/editor/defold_project.clj

```diff
@@ -839,25 +841,57 @@
         (settings-core/get-setting ["project" "dependencies"])
         (library/parse-library-uris))))
 
-(def ^:private embedded-resource? (comp nil? resource/proj-path))
+(defn- project-resource-node? [basis node-id]
+  (if-some [resource (resource-node/as-resource-original basis node-id)]
+    (some? (resource/proj-path resource))
+    false))
```

Comment:
The questing we're asking is, "Does this node-id represent an actual resource in the project?". I.e. it's a file or a zip entry from a dependency. We only want to consider the original nodes here, since override nodes are created when instantiating components, etc. These override nodes present as the instantiated files in every other way, but they're not.